noir_stdlib/src/hash.nr

```diff
@@ -246,3 +257,11 @@ impl<A, B, C, D, E> Hash for (A, B, C, D, E) where A: Hash, B: Hash, C: Hash, D:
         self.4.hash(state);
     }
 }
+
+#[test]
+fn assert_pedersen_noir() {
+    assert_eq(pedersen_hash_with_separator([1, 2, 3], 4), pedersen_hash_with_separator_noir([1, 2, 3], 4));
+    assert_eq(
+        pedersen_commitment_with_separator([1, 2, 3], 4), pedersen_commitment_with_separator_noir([1, 2, 3], 4)
+    );
+}
```

Comment:
Ah, just saw this. Can you have a version which takes some fields as arguments to improve coverage?

Reply:
Done